client/src/components/Verejne/index.js

```diff
@@ -42,35 +75,28 @@ const Verejne = ({
         itemClass="page-item"
         linkClass="page-link"
         hideNavigation
-        pageRangeDisplayed={pageCount}
+        pageRangeDisplayed={VEREJNE_PAGE_RANGE}
         activePage={currentPage}
         itemsCountPerPage={VEREJNE_MAX_PAGE_ITEMS}
         totalItemsCount={entitiesLength}
         onChange={setCurrentPage}
       />
     </div>
-    <GoogleMap />
+    <GoogleMap refetch={() => refetch(getEntitiesUrlFromMapReference(mapReference))} />
     <Legend />
   </div>
 )
 export default compose(
   connect(
     (state) => ({
-      entities: entitiesSelector(state),
-      entitiesLength: entitiesSelector(state).length,
+      entitiesLength: entitiesLengthSelector(state),
+      entities: currentPageEntities(state),
+      currentPage: currentPageSelector(state),
+      pageCount: pageCountSelector(state),
+      mapReference: mapReferenceSelector(state),
     }),
-    {fetchEntities}
+    {selectEntity, setCurrentPage}
```

Comment:
does mapDispatchToProps work this way ? (I'm kind of hoping this is some new syntax I don't know about where it takes dispatch from context, but probably just typo)